bindings_test.go

```diff
@@ -1151,6 +1167,20 @@ func testLOBRetrieval(t *testing.T, useArrowFormat bool) {
 	})
 }
 
+func TestMaxLobSizeSwitch(t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
+		dbt.exec(disableLargeVarcharAndBinary)
+		rows, err := dbt.query("select randstr(20000000, random())")
+		defer rows.Close()
+		assertStringContainsF(t, err.Error(), "Actual length 20000000 exceeds supported length")
+
+		dbt.exec(enableLargeVarcharAndBinary)
+		rows, err = dbt.query("select randstr(20000000, random())")
+		assertNilF(t, err)
+		dbt.exec(unsetLargeVarcharAndBinary)
```

Comment:
Shouldn't be `defer`red?

Reply:
what do you mean?

Reply:
Let's suppose you set this parameter above. Then, we reuse this session (because we have a pool). So if the test fail for instance a line above, this parameter will never be unset. Maybe we can add unsetting it as `defer` function at the beginning, wdyt?

Reply:
Makes sense, added defer for unsetting the param.